docker-compose/monitor/README.md

```diff
@@ -81,7 +81,7 @@ docker compose -f docker-compose-v2.yml up
 **Tips:**
 - Let the application run for a couple of minutes to ensure there is enough time series data to plot in the dashboard.
 - Navigate to Jaeger UI at http://localhost:16686/ and inspect the Monitor tab. Select `redis` service from the dropdown to see more than one endpoint.
-- To visualize the raw metrics stored on the Prometheus server (for debugging and local development use cases), use the built-in Prometheus UI at http://localhost:9090/query. For example, http://localhost:9090/query?g0.expr=traces_span_metrics_calls_total&g0.tab=0&g0.range_input=5m
+- To visualize the raw metrics stored on the Prometheus server (for debugging and local development use cases), use the built-in Prometheus UI at http://localhost:9090/query. For example, http://localhost:9090/graph?g0.expr=traces_span_metrics_calls_total&g0.tab=0&g0.range_input=5m
```

Comment:
does the `graph` endpoint still work? I thought it was replaced by the `query` endpoint.

Reply:
Query didn't work for me with the version of Prometheus that is invoked from docker compose file. Did you run a different version? 3.x has been released.

Reply:
Hm that's weird - I remember using `graph` was causing the CI to fail which was fixed in https://github.com/jaegertracing/jaeger/pull/6217